integration-tests/src/test/groovy/com/okta/sdk/tests/it/FactorsIT.groovy

```diff
@@ -48,7 +48,7 @@ class FactorsIT extends ITSupport {
 
     private String smsTestNumber = "185 635 15491"
 
-    @Test (groups = "group3")
+    @Test
```

Comment:
Some cells don't support SMS factor. Hence this test is removed from PDV test suite.